stories/Templates/Authors/Authors.jsx

```diff
@@ -1,18 +1,17 @@
-import React, { useEffect } from 'react';
-import logowhite from '../../assets/images/undp-logo-white.svg';
-import logo from '../../assets/images/undp-logo-blue.svg';
-import user from '../../assets/images/user.svg';
-import viewport from '../../assets/js/viewport';
-import { Heading } from '../../Atom/Typography/Heading/Heading';
-import { Breadcrumbcomponent } from '../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs';
-import GlobalHeader from '../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader';
-import { Footer } from '../../Organism/Footer/Footer';
-import './Authors.scss';
-import init from '../../assets/js/select-init';
-import { Authorcard } from '../../Components/UIcomponents/Author/Authorcard/Authorcard';
-import { CtaButton } from '../../Components/UIcomponents/Buttons/Cta_button/CtaButton';
-import { SearchExpand } from '../../Components/Forms/expandable search/SearchExpand';
-import CustomSelect from '../../Components/Forms/Dropdowns/CustomSelect/CustomSelect';
+import React, { useEffect } from "react";
+import logowhite from "../../assets/images/undp-logo-white.svg";
+import logo from "../../assets/images/undp-logo-blue.svg";
+import user from "../../assets/images/user.svg";
+import viewport from "../../assets/js/viewport";
+import { Heading } from "../../Atom/Typography/Heading/Heading";
+import { Breadcrumbcomponent } from "../../Components/Navigationcomponents/Breadcrumbs/Breadcrumbs";
+import GlobalHeader from "../../Components/Navigationcomponents/Mainnavigation/GlobalHeader/GlobalHeader";
+import { Footer } from "../../Organism/Footer/Footer";
+import "./Authors.scss";
+import init from "../../assets/js/select-init";
+import { Authorcard } from "../../Components/UIcomponents/Author/Authorcard/Authorcard";
+import { CtaButton } from "../../Components/UIcomponents/Buttons/Cta_button/CtaButton";
+import { SearchExpand } from "../../Components/Forms/expandable search/SearchExpand";
```

Comment:
@Ashish-srij code should be linted as it is showing double quotes instead of single quotes